docs/reference/commandline/tag.md

```diff
@@ -35,7 +35,7 @@ components may contain lowercase letters, digits and separators. A separator
 is defined as a period, one or two underscores, or one or more dashes. A name
 component may not start or end with a separator.
 
-A tag name must be valid ASCII and may contain lowercase and uppercase letters,
+A tag name must be valid ASCII and may contain lowercase letters,
```

Comment:
Looks like it's supported;

```bash
$ docker tag busybox foo:ONE

$ docker images

REPOSITORY  TAG   IMAGE ID            CREATED             SIZE
foo         ONE   00f017a8c2a6        2 months ago        1.11MB
```


And looking in that same file; I _think_ this is the RegEx used for that;
https://github.com/docker/distribution/blob/95daa793b83a21656fe6c13e6d5cf1c3999108c7/reference/regexp.go#L36-L37

Did you run into this issue?